build.sbt

```diff
@@ -22,9 +22,13 @@ libraryDependencies ++= Seq(
   "org.slf4j" % "slf4j-api" % "1.7.2",
   "org.slf4j" % "slf4j-log4j12" % "1.7.2",
   "org.scalatest" %% "scalatest" % "1.9.1" % "test",
+  "org.mockito" % "mockito-core" % "1.8.5",
```

Comment:
I haven't used Mockito personally, but it looks like this is what Spark is using for mock tests these days, so I'm good with adding this as a dependency.